Dockerfile

```diff
@@ -110,7 +111,6 @@ USER root
 
 COPY --from=back-end-builder /kargo/bin/ /usr/local/bin/
 COPY --from=tools /tools/ /usr/local/bin/
-COPY --from=ui-builder /ui/build /ui/build
```

Comment:
No need to copy these files over anymore, because they're already baked into the binary.